Dockerfile

```diff
@@ -1,21 +1,20 @@
-FROM python:3.6-alpine
+FROM python:3.6-jessie
 
 WORKDIR /app/woeip
 
 ENV DJANGO_SETTINGS_MODULE woeip.settings
 ENV PIPENV_DONT_USE_PYENV 1
 ENV PIPENV_SYSTEM 1
 
-RUN apk add --update \
-    coreutils \
-    gcc \
-    libffi-dev \
-    make \
-    musl-dev \
-    postgresql-dev \
-    python3-dev \
-  && pip install pipenv \
-  && rm -rf /var/cache/apk/*
+RUN apt-get update && \
```

Comment:
The bash commands are definitely welcome sight. I'm more familiar with debian/ubuntu. An unintended consequence of switching from alpine is that its making it easier for me to decode how this all works.